qutebrowser/browser/commands.py

```diff
@@ -992,7 +992,7 @@ def tab_move(self, index: typing.Union[str, int]=None, count=None):
             self._tabbed_browser.setUpdatesEnabled(True)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
-                       maxsplit=0)
+                       maxsplit=0, no_replace_variables=True)
```

Comment:
I removed maxsplit and no_replace_variables to let the regular parser handle things.